library/network_connections.py

```diff
@@ -805,6 +812,11 @@ def connection_create(self, connections, idx, connection_current=None):
             )
             s_wired = self.connection_ensure_setting(con, NM.SettingWired)
             s_wired.set_property(NM.SETTING_WIRED_MAC_ADDRESS, connection["mac"])
+            if hasattr(NM, "SettingMatch") and hasattr(NM, "SETTING_MATCH_PATH"):
```

Comment:
This will silently ignore `interface_path` if NM does not support this.

Reply:
I think we should raise an error when `connection["interface_path"] != []` and when 
`if hasattr(NM, "SettingMatch") and hasattr(NM, "SETTING_MATCH_PATH")` evaluates to false

Reply:
I think we should raise an error when `connection["interface_path"] != []` and when 
`if hasattr(NM, "SettingMatch") and hasattr(NM, "SETTING_MATCH_PATH")` evaluates to false

Reply:
yes, I think you can do `if connect["interface_path"] ...` since the empty list is considered to be False.

Reply:
checking support should be done in `validate_connection_one()`. In general, `ArgValidator` is independent of the used provider. Only at a later step (`validate_connection_one()`), the settings are validated w.r.t. the provider.

Also, for initscripts, this parameter must be rejected too.

Reply:
moved the validation part into the `validate_connection_one()`, see
https://github.com/linux-system-roles/network/blob/aa27d56c6d67d7b883b5415a3a972678642ab835/module_utils/network_lsr/argument_validator.py#L1823